conf/obj_ops.c

```diff
@@ -172,6 +172,9 @@ M0_INTERNAL void m0_conf_obj_delete(struct m0_conf_obj *obj)
 	M0_PRE(obj->co_status != M0_CS_LOADING);
 
 	/* Finalise generic fields. */
+	/* XXX: spiel-conf-ut:create-fail fails here
```

Comment:
Is this comment still needed, as i don't see this UT failure.

Reply:
Ah, yes, this is not needed.
I'll remove this comment.

Reply:
Done.